src/transformers/models/emformer/modeling_emformer.py

```diff
@@ -0,0 +1,1262 @@
+# coding=utf-8
+# Copyright 2022 The TorchAudio authors and The HuggingFace Inc. team,
```

Comment:
@sgugger FYI we had to use a different license here (as requested by authors) - but should be fine no?

Reply:
I am not a lawyer, so I have no idea :-)

Reply:
Haha fair point - sorry unnecessary ping here (just a FYI then ;-))